lib/ttt_app.rb

```diff
@@ -0,0 +1,11 @@
+module TttApp
+  class Error < StandardError; end
+  # Your code goes here...
+  print('Welcom To TTT')
+end
+
+class TicTacToe
+  def self.printWelcome
```

Comment:
Method names are in snake case in Ruby, so this should be `self.print_welcome`.

Reply:
okay